modules/hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns_{context}"]
+= Configuring DNS
```

Comment:
It is not clear to me what this procedure does. 
Again, must I do this, if so in what cases?

Reply:
@mburke5678 we have now defined the above section. Request you to take a look. Thanks.